docs/src/model_components/biogeochemical/NPZ.md

```diff
@@ -2,7 +2,7 @@
 
 The provided NPZD model is the low complexity model of [Kuhn2015](@citet).
 
-![Diagram of NPZD formulation](npzd.svg){width = "700px"}
+![Diagram of NPZD formulation](npzd.svg)
```

Comment:
We should be able to set the size of the image. Maybe @navidcy knows how to do that?

Reply:
Maybe https://github.com/JuliaDocs/Documenter.jl/issues/423 still works